eng/templates/official/jobs/ci-e2e-tests.yml

```diff
@@ -79,6 +79,65 @@ jobs:
           python setup.py extension
           mkdir logs
         displayName: 'Install dependencies and the worker'
+        condition: and(eq(variables['USETESTPYTHONSDK'], false), eq(variables['USETESTPYTHONEXTENSIONS'], false))
+      - task: DownloadPipelineArtifact@2
+        displayName: 'Download Python SDK Artifact'
+        inputs:
+          buildType: specific
+          artifactName: 'azure-functions'
+          project: '3f99e810-c336-441f-8892-84983093ad7f'
```

Comment:
Does this have to be project ID or does project name work too?

Reply:
If name does not work can you add a comment with the name so that we know where to check if there is any issue